src/models/common/index.ts

```diff
@@ -11,10 +11,15 @@ export interface IssuedCurrency {
   issuer: string
 }
 
+export interface Balance {
+  currency: string
+  value: string
+}
+
```

Comment:
Whta's a balance?

Reply:
I made up a name for the object, happy to change it/move it. This type is used by `gateway_balances` and probably nowhere else.

Reply:
if it's only used by `GatewayBalances` then shouldn't it be defined in some `gatewayBalances.ts` file and _not_ exported?